### PR TITLE
New scouting queue BuildPool.Windows.VS2019.Pre.Scouting.Open

### PR DESCRIPTION
**Pull request by lukas-lansky**
Windows 10 Bring-Your-Own-Cloud AMD64 build queue with VS 2019 Preview intended for scouting

### CODE REVIEW

File: src/appsettings.dncengpublic-prod.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
     "BuildPool.Windows.Amd64.VS2019.Pre.ES.Open",
     "BuildPool.Windows.10.Amd64.VS2019.BT.Open",
     "BuildPool.Windows.10.Wasm.Open",
-    "BuildPool.Windows.10.Amd64.VS2019.tmp.Open"
+    "BuildPool.Windows.10.Amd64.VS2019.tmp.Open",
```
Comment (MattGal) [inline src/appsettings.dncengpublic-prod.json:29]: If this one's deprecated, do we want to delete it?

Comment (lukas-lansky) [reply src/appsettings.dncengpublic-prod.json]: If I understood correctly, we deprecated that name and are going to call it `BuildPool.Windows.VS2019.Scouting.Open`?

**Comment by riarenas** [comment]
This should go in tomorrow's rollout correct? @lukas-lansky is it ready to merge?

**Comment by lukas-lansky** [comment]
Yes, it is. Thanks @riarenas!

**Pull request merged**